pkg/boot/align/align.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015-2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package align
```

Comment:
why is this in its own package?

Reply:
I want to move object.go to be under Linux, and it need to call Align up by page size, which is a private func and used in kexec memory.